core/scripts/umip-3/2_VoteSimulate.js

```diff
@@ -153,14 +161,14 @@ async function runExport() {
     (await voting.getCurrentRoundId()).toString()
   );
 
-  assert.equal((await voting.getPendingRequests()).length, 0); // There should be no pending requests as vote is concluded
+  assert.equal((await voting.getPendingRequests()).length, 1); // There should be no pending requests as vote is concluded
 
   /** *******************************************************************
    * 4) Execute proposal submitted to governor now that voting is done *
    **********************************************************************/
 
   console.log("4. EXECUTING GOVERNOR PROPOSALS");
-  const proposalId = (await governor.numProposals()).subn(1).toString(); // most recent proposal in voting.sol
+  const proposalId = (await governor.numProposals()).subn(2).toString(); // most recent proposal in voting.sol
```

Comment:
Same comment

Reply:
@chrismaree just in case you missed this

Reply:
ye it's correct now I think :) the problem is that this script will need to be updated according to the specific vote we are acting on and as such the as the DVM's state due to proposals/votes etc it'll need to be adapted accordingly.

Reply:
Yeah which makes this less useful for the future. Is there any way to read it dynamically

Reply:
You can at least make this one less than the number hardcoded into line 47.

```
const startingProposals = 2
assert.equal(...numProposals.length, startingProposals+1)
const proposalId = (...numProposals()).subn(startingProposals).toString()
```